plugins/catalog-backend-module-gitlab-org/src/catalogModuleGitlabOrgDiscoveryEntityProvider.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import {
+  coreServices,
+  createBackendModule,
+} from '@backstage/backend-plugin-api';
+import { GitlabOrgDiscoveryEntityProvider } from '@backstage/plugin-catalog-backend-module-gitlab';
```

Comment:
_a question for the maintainers; also rather for a follow-up_

I know that this follows the same pattern as used for the GitHub org provider. However, I have the same question there.

As we create a separate module and package for `gitlab-org`, wouldn't it be better to move the provider to this package and deprecate it at the other one (e.g., by importing it from this new package and reexporting it, marked as deprecated -- similar to few other places).

What do you think?

Reply:
Hi @pjungermann ,

 I would be glad to do it if provided with an example! :)

Reply:
@vinzscam yeah, nothing that blocks this PR.

I'm also not sure what the maintainers opinion is on that one. I think it would make sense to split that up.
It does affect other plugins like the one for github-org as well.

Reply:
I think the issue is that `gitlab` can't import from `gitlab-org`, as this will cause a circular dependency